vm_supervisor/storage.py

```diff
@@ -155,28 +161,93 @@ def create_ext4(path: Path, size_mib: int) -> bool:
     return True
 
 
+async def create_devmapper(volume: PersistentVolume | RootfsVolume, namespace: str) -> Path:
+    volume_name = volume.name if isinstance(volume, PersistentVolume) else "rootfs"
+    volume_dev_name = f"{namespace}_{volume_name}"
+    path_volume_dev_name = Path(f"/dev/mapper/{volume_dev_name}")
+    if path_volume_dev_name.is_block_device():
+        return path_volume_dev_name
+    path = Path(
+        join(settings.PERSISTENT_VOLUMES_DIR, namespace, f"{volume_name}.ext4")
+    )
```

Comment:
Avoid mixing `os.path` and `pathlib`.

```suggestion
    path = Path(settings.PERSISTENT_VOLUMES_DIR) / namespace / f"{volume_name}.ext4"
```